src/v2/.eslintrc.yml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
why?

Reply:
We move the JS generated code under `build/`, which changes where it requires `package.json`.

Without this, `eslint` complains it cannot require `package.json`

Reply:
oh yeah!